src/components/ErrorList/ErrorList.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames';
+import React, { useEffect } from 'react';
+
+interface Props {
+  errors: string[];
+  onClear: () => void;
+}
+
+export const ErrorList: React.FC<Props> = ({ errors, onClear }) => {
+  useEffect(() => {
+    if (errors.length !== 0) {
```

Comment:
Replace it with:

`errors.length > 0`
or
`errors.length`

It will be the same.